spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -195,70 +200,128 @@
+    after do
+      reload_ab_tests
+    end
 
-      expect(page).not_to have_content(t('doc_auth.tips.document_capture_selfie_text1'))
-      attach_passport_image(
+    context 'with a valid passport' do
+      let(:passport_image) do
         Rails.root.join(
           'spec', 'fixtures',
           'passport_credential.yml'
-        ),
-      )
+        )
+      end
 
-      submit_images
-      expect(page).to have_content(t('doc_auth.headings.capture_complete'))
-    end
-  end
+      it 'happy path' do
+        choose_id_type(:passport)
+        expect(page).to have_content(t('doc_auth.headings.document_capture_passport'))
+        expect(page).to have_current_path(idv_document_capture_url)
 
-  context 'with an invalid passport', allow_browser_log: true do
-    let(:fake_dos_api_endpoint) { 'http://fake_dos_api_endpoint/' }
+        expect(page).not_to have_content(t('doc_auth.tips.document_capture_selfie_text1'))
+        attach_passport_image(passport_image)
+        submit_images
+        expect(page).to have_content(t('doc_auth.headings.capture_complete'))
+        fill_out_ssn_form_ok
+        click_idv_continue
+        expect_step_indicator_current_step(t('step_indicator.flows.idv.verify_info'))
+        expect(page).to have_content(t('doc_auth.headings.address'))
+        fill_in 'idv_form_address1', with: '123 Main St'
+        fill_in 'idv_form_city', with: 'Nowhere'
+        select 'Virginia', from: 'idv_form_state'
+        fill_in 'idv_form_zipcode', with: '66044'
+        click_button t('forms.buttons.submit.update')
+        expect(page).to have_current_path(idv_verify_info_path)
+        expect(page).to have_content('VA')
+        expect(page).to have_content('123 Main St')
+        expect(page).to have_content('Nowhere')
+        complete_verify_step
+        expect(page).to have_current_path(idv_phone_url)
+      end
+    end
 
-    before do
-      stub_request(:post, fake_dos_api_endpoint)
-        .to_return(status: 200, body: '{}', headers: {})
+    context 'with an invalid passport' do
+      let(:passport_image) do
+        Rails.root.join(
+          'spec', 'fixtures',
+          'passport_bad_mrz_credential.yml'
+        )
+      end
 
-      allow(IdentityConfig.store).to receive(:dos_passport_mrz_endpoint)
-        .and_return(fake_dos_api_endpoint)
-      visit_idp_from_oidc_sp_with_ial2
-      sign_in_and_2fa_user(@user)
-      complete_doc_auth_steps_before_document_capture_step
+      it 'fails due to mrz' do
+        choose_id_type(:passport)
+        expect(page).to have_current_path(idv_document_capture_url)
+        expect(page).not_to have_content(t('doc_auth.tips.document_capture_selfie_text1'))
+        attach_passport_image(passport_image)
+        submit_images
+        expect(page).not_to have_content(t('doc_auth.headings.capture_complete'))
+        expect(page).to have_content('invalid MRZ')
+        expect_to_try_again
+        expect(page).to have_content('invalid MRZ')
```

Comment:
We'll have to update this once #12094 is merged since the error is changing to an i18n'd message.